src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -131,17 +131,17 @@ public void stubOutSamCalls(IamProviderInterface samService) throws Exception {
         // in the bookkeeping lists (createdDatasetIds/createdDatasetIds) in this class.
         when(samService.listAuthorizedResources(any(), eq(IamResourceType.DATASET)))
             .thenAnswer((Answer<List<UUID>>) invocation
-                -> createdDatasetIds.stream().map(UUID::fromString).collect(Collectors.toList()));
+                -> new ArrayList<>(createdDatasetIds));
```

Comment:
I don't think we ned to wrap this in a new ArrayList, right?

Reply:
Good point!